server/api.go

```diff
@@ -286,6 +287,33 @@ func (p *Plugin) handleDebug(w http.ResponseWriter, r *http.Request) {
 	res.code = http.StatusNotFound
 }
 
+// handleConfig returns the client configuration, and cloud license information
+// that isn't exposed to clients yet on the webapp
+func (p *Plugin) handleConfig(w http.ResponseWriter) error {
+	license := p.pluginAPI.System.GetLicense()
+	if license == nil {
+		p.handleErrorWithCode(w, http.StatusBadRequest, "no license",
+			errors.New("no license found"))
+		return nil
+	}
+
+	clientConfig := p.getConfiguration().getClientConfig()
+	config := map[string]interface{}{
+		"ice_servers":            clientConfig.ICEServers,
+		"allow_enable_calls":     clientConfig.AllowEnableCalls,
+		"default_enabled":        clientConfig.DefaultEnabled,
```

Comment:
Start moving over to using json-standard snake case

Reply:
This is a breaking change though right? Are you thinking of handling it on mobile somehow or dropping support?

Reply:
Right, I must have blanked on that. Fixed.